docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.8'
+networks:
+  db-data-net:
+    driver: bridge
+
+services:
+  pythonapp:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8081:8080"
+    environment:
+      ENV: local
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+
+  mysql:
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+
+    container_name: mysql
+    ports:
+      - "3306:3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: 1234
```

Comment:
The `MYSQL_ROOT_PASSWORD` is set to a weak password ('1234'). It's important to use a strong, secure password to protect your database from unauthorized access.